apps/web/src/components/content/sections/unified-section.tsx

```
@@ -78,39 +96,141 @@ function Wrapper({
+    }
+  }, [html]);
+
+  // Early return check after hooks (Rules of Hooks compliance)
+  if (!html || typeof html !== 'string') {
+    return <div className={className} />;
+  }
+
+  // During SSR, render unsanitized (will be sanitized on client hydration)
+  const htmlToRender = isClient ? safeHtml : html;
+
+  return <div className={className} dangerouslySetInnerHTML={{ __html: htmlToRender }} />;
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)